test/james.crypto.md5.tests.pas

```diff
@@ -28,10 +28,16 @@
 interface
 
 uses
-  Classes, SysUtils, fpcunit, testregistry,
+  Classes, SysUtils,
   James.Data,
   James.Data.Clss,
-  James.Crypto.MD5.Clss;
+  James.Crypto.MD5.Clss,
+  james.testing.clss,
+  {$IFDEF FPC}
+    james.testing.clss.fpc
+  {$ELSE}
+    james.testing.clss.delphi
+  {$ENDIF};
```

Comment:
@nunopicado we need to do this https://github.com/mdbs99/james/pull/62/commits/3b9bf5ffd18d75244fc99ca7bd2db52c1201a868#r128682154 to refactor all tests, removing these IFDEFs.